src/bin/travis/resources/conformance_expected_failures.txt

```diff
@@ -49,7 +49,7 @@
 #49
 #50
 51
-52
+#52
```

Comment:
Interesting... I had this test go green for me too on Local by just fully qualifying the value for the `path` attribute. 🤔